pages/pipelines/trigger_step.md.erb

```diff
@@ -206,3 +206,31 @@ To set environment variables on the build created by the trigger step, use the `
       RELEASE_STREAM: "${RELEASE_STREAM:-stable}"
 ```
 {: codeblock-file="pipeline.yml"}
+
+## Triggering specific steps in a pipeline
+
+While you cannot trigger only a specific step in a pipeline, you can use [conditionals](/docs/pipelines/conditionals) achieve a similar effect. A combination of the following environment variables might be useful:
+
+* Testing for [BUILDKITE_SOURCE](/docs/pipelines/environment-variables#) to find out if the build was triggered by a trigger step
```

Comment:
Might be worth explicitly calling out that this value is set to `trigger_job` for trigger jobs, just to make sure the presence of it doesn't indicate that the job source was a trigger job.